web/viewer.js

```diff
@@ -1873,12 +1873,12 @@ window.addEventListener('pagechange', function pagechange(evt) {
   }
 }, true);
 
+var zoomDisabled = false, zoomDisablerId;
```

Comment:
Nit: `zoomDisablerId` seems to me like a slightly strange name, how about `zoomDisabledTimeout` instead?